content/chat/patterns/storage.textile

```diff
@@ -0,0 +1,190 @@
+---
+title: Message storage
```

Comment:
Given we are talking generally about all integrations, I think we should keep this title generic too. wdyt @m-hulbert?